### ISSUE TITLE
VGG.npy

### ISSUE THREAD

**Issue by ximi2018** [yangerkun/IJCAI2018_SSDH#3]
where is the vgg,npy?

**Comment by yangerkun** [comment]
You can download the vgg16.npy from here : https://mega.nz/#!YU1FWJrA!O1ywiCS2IiOlUCtCpI6HTJOMrneN-Qdv3ywQP5poecM

**Comment by ximi2018** [comment]
Hello, there is no image_train.txt file in the mirflickr dataset I downloaded. Can you give me a nuswide and mirflickr link

**Comment by yangerkun** [comment]
> Hello, there is no image_train.txt file in the mirflickr dataset I downloaded. Can you give me a nuswide and mirflickr link

As indicated by the Readme, image_train.txt only contains the image path and labels. I think you can create it easily.

**Comment by ximi2018** [comment]
But the label and path in the code seem to be divided into two txt files

[](![image](https://user-images.githubusercontent.com/44973744/68096703-91e63c00-feed-11e9-839c-6f405ea581a6.png))

**Comment by yangerkun** [comment]
> But the label and path in the code seem to be divided into two txt files

Yes, I check the code and they should be indicated in different files. I will correct it.

**Comment by ximi2018** [comment]
Hi, I reported a list index out of range error when I ran the code,           
mxnet_image_data.py   line 77
 self._label[i] = [int(j) for j in self.l_lines[i].strip().split()[:]] 
test_Label.txt has 1000 lines
How to solve it
